package.json

```diff
@@ -12,7 +13,9 @@
     "test:watch": "npm run test:unit-source -- --watch",
     "test:update": "npm run test:unit-source -- --updateSnapshot",
     "build": "npm run build:js && npm run build:css",
+    "build:watch": "npm run build:js -- --watch | npm run build:css -- --watch",
```

Comment:
Does the `--watch` flag work on the css build? I'll be damned!

Reply:
nope, forgot to remove that 😀